Issue #2640: Change abbreviationAsWordInName error message to contain…

liscju opened the pull request:
…s name that violates a format

Change error msg as issue requested. I added in com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheckTest helper method getWarningMessage which is nearly same as introduced getWarningMessage in com.google.checkstyle.test.chapter5naming.rule53camelcase.AbbreviationAsWordInNameCheckTest and they differs by first argument(class). 

Why there are in two different packages test classes which have same purpose? If this is not intended i could merge them in this PR.

rnveach commented:
> Why there are in two different packages test classes which have same purpose? 

If you mean "it" versus "test" folder, it is because "it" validates and proves that we live up to just google's style of coding alone using google's configuration, and "test" validates every inch of checkstyle and all its properties regardless of google/sun/custom styles.

liscju commented:
@rnveach you are right, i have no idea why i didn't notice "it" in path :P

romani commented:
TC is not happy, need to be fixed too

romani commented:
@liscju, please update properies of all other languages.

liscju commented:
@romani tc is required me to log in to see its report, are there a general account for checkstyle contributors to see it, or there are any other way to generate it?

rnveach commented:
@liscju When you are at the login screen, to the right of the "Log in" button is a "Log in as guest" link. Once in, click "View code inspection report" and it will show you the error, which is:

> src/test/java/com/puppycrawl/tools/checkstyle/checks/naming
> AbbreviationAsWordInNameCheckTest.java
> line 35: `warningMessage` Field can be converted to a local variable

liscju commented:
@romani 
1) fixed issue with TC but im still waiting for result
2) i updated properties for other languages as well, unfortunately i dont know those languages so i had to do this with google translator, so i can only hope it is correct :P

romani commented:
merged as FF, thanks a lot !

>  i had to do this with google translator, so i can only hope it is correct

it is ok, we did the same, when filled missed translations.
